docs/docs/Agents/agents.mdx

```diff
@@ -94,17 +94,14 @@ For a multi-agent example, see [Use an agent as a tool](/agents-tools#use-an-age
 
 You can configure the **Agent** component to use your preferred provider and model, custom instructions, and tools.
 
-:::tip
 Many optional **Agent** component input parameters are hidden by default in the visual editor.
 You can access all component parameters through the <Icon name="SlidersHorizontal" aria-hidden="true"/> **Controls** in the [component's header menu](/concepts-components#component-menus).
-:::
-
 ### Provider and model
 
 Use the **Model Provider** (`agent_llm`) and **Model Name** (`llm_model`) settings to select the model provider and LLM that you want the agent to use.
 
 The **Agent** component includes many models from several popular model providers.
-To access other providers and models, set **Model Provider** to **Custom**, and then connect any [**Language Model** component](/components-models).
+To access other providers and models, set **Model Provider** to <Icon name="Plus" aria-hidden="true"/> **Connect Other Models**, and then connect any [**Language Model** component](/components-models).
```

Comment:
I am hesitant to merge this PR into doc-1.6 until we see this in the release qa build.

Reply:
Sure, it can wait.